assets/js/modules/analytics-4/components/settings/SettingsView.stories.js

```diff
@@ -121,11 +121,10 @@ export default {
 
 				registry
 					.dispatch( MODULES_ANALYTICS_4 )
-					.setEnhancedMeasurementStreamEnabled( {
-						propertyID,
-						webDataStreamID,
-						enabled: true,
-					} );
+					.receiveGetEnhancedMeasurementSettings(
+						fixtures.defaultEnhancedMeasurementSettings,
+						{ propertyID, webDataStreamID }
+					);
```

Comment:
This was necessary to fix a failure in VRT due to the added resolver for `isEnhancedMeasurementStreamEnabled`. `setEnhancedMeasurementStreamEnabled` is incorrect to use here – we should be pre-receiving the state which also happens to have `streamEnabled: true` in the fixture.